packages/plugin-conversation/src/conversation.js

```diff
@@ -460,6 +460,34 @@ const Conversation = SparkPlugin.extend({
   },
 
   /**
+   * Assigns an avatar to a room
+   * @param {Object} conversation
+   * @param {File} avatar
+   * @returns {Promise<Activity>}
+   */
+  assign(conversation, avatar) {
+    if ((avatar.size || avatar.length) > 1024 * 1024) {
+      return Promise.reject(new Error(`Room avatars must be less than 1MB`));
+    }
+    return this._inferConversationUrl(conversation)
+      .then(() => {
+        const activity = ShareActivity.create(conversation, null, this.spark);
+        activity.enableThumbnails = false;
```

Comment:
Now that you say that, yeah it would work but it probably makes things confusing.